charts/rancher-monitoring/v0.1.3/questions.yml

```diff
@@ -1 +1,2 @@
-rancher_min_version: 2.4.9-rc1
+rancher_min_version: 2.4.9-rc0
+rancher_max_version: 2.4.9-rc0
```

Comment:
Weird, this one was never released to 2.4.x line. Its 2.5.0 in release branch, then got bumped to 2.4.9 recently here. So I guess this makes sense, we'll never release 0.1.3 to 2.4.x line. 
* 2.4.8 will have 0.1.2
* 2.4.9 will have 0.1.4